aiida/manage/external/rmq/launcher.py

```diff
@@ -9,8 +9,6 @@
 
 LOGGER = logging.getLogger(__name__)
 
-__all__ = ('ProcessLauncher',)
```

Comment:
NOTE: This is a breaking API change in principle, but since this only seems to be used in `aiida/manage/manager.py` I think it is okay. This is AFAIK the only way how to avoid importing `plumpy` upon every import, since the `ProcessLauncher` uses `plumpy.ProcessLauncher` as a parent class.